python/pyrogue/_Device.py

```diff
@@ -526,6 +527,28 @@ def _decorator(func):
             return func
         return _decorator
 
+class ArrayDevice(Device):
```

Comment:
Can you drop an example in the doc section, maybe in a future notes directory.

Reply:
I added some examples in the PR description. I'll do it in the documentation once we merge it.
I'm open to discussion on this implementation. I wanted a quick way to group a bunch of nearly identical Nodes into a parent, but maybe this makes the code harder read and follow.